github_connector_odoo/views/view_odoo_license.xml

```diff
@@ -76,7 +76,7 @@ License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
                     <t t-name="kanban-box">
                     <div class="oe_module_vignette oe_kanban_global_click">
                         <t t-if="record.image.raw_value">
-                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt=""/>
+                            <img t-att-src="'data:image/png;base64,'+record.image.raw_value" class="oe_module_icon" alt="${record.image.name}"/>
```

Comment:
Not valid on QWeb. You have to use `t-att-alt="variable"`, but `record.image.name` is not a valid variable. I would say `record.name.raw_value` if the model has a field name and it's included in the view. `raw_value` is due to this being a JS QWeb view.